x/shield/keeper/test_common.go

```diff
@@ -82,21 +82,21 @@ func RandomPoolInfo(r *rand.Rand, k Keeper, ctx sdk.Context) (uint64, string, bo
 }
 
 // RandomCollateral returns a random collateral given access to the keeper and ctx.
-func RandomCollateral(r *rand.Rand, k Keeper, ctx sdk.Context) (types.Collateral, bool) {
+func RandomCollateral(r *rand.Rand, k Keeper, ctx sdk.Context) (uint64, types.Collateral, bool) {
```

Comment:
why return poolID?

Reply:
Forgot we have poolID in collateral. I'll remove it.